projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/components/query-write/query-write-raise-query/query-write-raise-query.component.html

```diff
@@ -10,98 +10,112 @@ <h1 class="govuk-heading-l govuk-!-margin-bottom-4">{{ 'Enter query details' | r
+            <label class="govuk-label govuk-label--m govuk-!-font-weight-bold">
+              {{ 'Is the query hearing related?' | rpxTranslate }}
+            </label>
+          </legend>
+          <div class="govuk-radios govuk-radios--inline" data-module="govuk-radios">
+            <div class="govuk-radios__item">
+              <input class="govuk-radios__input" id="query-raise-isHearingRelated-yes" name="isHearingRelated"
+                     type="radio"
+                     formControlName="isHearingRelated"
+                     [checked]="formGroup.get('isHearingRelated').value === true" [value]="true"
+              >
+              <label class="govuk-label govuk-radios__label" for="query-raise-isHearingRelated-yes">
+                {{ 'Is the query hearing related?' | rpxTranslate: null : 'Yes' }}
+              </label>
+            </div>
+            <div class="govuk-radios__item">
+              <input class="govuk-radios__input" id="query-raise-isHearingRelated-no" name="isHearingRelated"
+                     type="radio"
+                     formControlName="isHearingRelated"
+                     [checked]="formGroup.get('isHearingRelated').value === false" [value]="false"
+              >
+              <label class="govuk-label govuk-radios__label" for="query-raise-isHearingRelated-no">
+                {{ 'Is the query hearing related?' | rpxTranslate: null : 'No' }}
               </label>
-            </legend>
+            </div>
+          </div>
+        </fieldset>
 
-            <div class="govuk-radios govuk-radios--inline" data-module="govuk-radios">
-              <div class="govuk-radios__item">
-                <input class="govuk-radios__input" id="query-raise-isHearingRelated-yes" name="query-raise-isHearingRelated-yes" type="radio"
-                       [checked]="formGroup.get('isHearingRelated').value === true" [value]="true"
-                >
-                <label class="govuk-label govuk-radios__label" for="query-raise-isHearingRelated-yes">
-                  {{ 'Is the query hearing related?' | rpxTranslate: null : 'Yes' }}
-                </label>
-              </div>
-              <div class="govuk-radios__item">
-                <input class="govuk-radios__input" id="query-raise-isHearingRelated-no" name="query-raise-isHearingRelated-no" type="radio"
-                       [checked]="formGroup.get('isHearingRelated').value === false" [value]="false"
-                >
-                <label class="govuk-label govuk-radios__label" for="query-raise-isHearingRelated-no">
-                  {{ 'Is the query hearing related?' | rpxTranslate: null : 'No' }}
+        <ng-container *ngIf="formGroup.get('isHearingRelated').value">
+          <div class="govuk-!-margin-top-1">
+            <fieldset class="govuk-fieldset" role="group">
+              <legend>
+                <label class="govuk-label govuk-label--s">
+                  {{ 'Hearing Date' | rpxTranslate }}
```

Comment:
@MateiHMCTS - just to check whether "Date" should be "date", given "Hearing" already has an initial capital

Reply:
Made the change